DayrSuryan/DSEthiop7.xml

```diff
@@ -46,7 +46,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <locus from="1r" to="51r" cert="medium"/>
```

Comment:
cert="medium" does actually not give very meaningful information...maybe you could phrase the uncertainty better in a note?

Reply:
solved